packages/flutter_tools/lib/src/android/java.dart

```diff
@@ -149,7 +149,7 @@ class Java {
     final String longVersionText = versionLines.length >= 2 ? versionLines[1] : versionLines[0];
 
     // The contents that matter come in the format '11.0.18' or '1.8.0_202'.
```

Comment:
Should this code comment be updated with more examples? This is just a nit, because anyone reading this code could look at the tests to learn what this is meant to match.

Reply:
Will update before merging